tests/integration/records/single_db/test_records_load_datetime.py

```diff
@@ -0,0 +1,174 @@
+                addl_hints = {
+                    'datetimeformat': f"{dateformat} HH:MI:SS",
+                    'datetimeformattz': f"{dateformat} HH:MI:SSOF",
+                    'timeonlyformat': "HH:MI:SS",
+                }
+            self.load(hint_name='dateformat',
+                      format_string=dateformat,
+                      column_name='date',
+                      field_type='date',
+                      # ensure a Pandas-compatible format in case
+                      # database doesn't support hints directly
+                      addl_hints=addl_hints)
+            date = self.pull_result(column_name='date')
+            if isinstance(date, str):
+                # If Pandas was required, the date type will be
+                # simplified to string as pandas doesn't support
+                # date-only columns
+                self.assertEqual(date, create_sample(dateformat),
+                                 f"Problem loading date with dateformat {dateformat}")
+            else:
+                self.assertEqual(date.year, SAMPLE_YEAR)
+                self.assertEqual(date.month, SAMPLE_MONTH)
+                self.assertEqual(date.day, SAMPLE_DAY)
+
+    def database_has_no_time_type(self) -> bool:
+        return self.engine.name == 'redshift'
+
+    def test_load_timeonly(self) -> None:
+        for timeformat in TIMEONLY_CASES:
+            self.load(hint_name='timeonlyformat',
+                      format_string=timeformat,
+                      column_name='time',
+                      field_type='time')
+            time = self.pull_result(column_name='time')
+            if self.database_has_no_time_type():
+                self.assertEqual(time, create_sample(timeformat))
+            elif self.database_provides_times_as_timedeltas():
+                assert isinstance(time, datetime.timedelta)
+                minutes = SAMPLE_HOUR*60 + SAMPLE_MINUTE
+                if 'SS' in timeformat:
+                    seconds = minutes*60 + SAMPLE_SECOND
+                else:
+                    seconds = minutes*60
+                self.assertEqual(time.seconds, seconds)
+
+            else:
+                self.assertEqual(time.hour, SAMPLE_HOUR)
+                self.assertEqual(time.minute, SAMPLE_MINUTE)
+                if 'SS' in timeformat:
+                    self.assertEqual(time.second, SAMPLE_SECOND)
+                else:
+                    self.assertEqual(time.second, 0)
+
+    def test_load_datetimetz(self) -> None:
+        for datetimetzformat in DATETIMETZ_CASES:
+            self.load(hint_name='datetimeformattz',
+                      format_string=datetimetzformat,
+                      column_name='datetimetz',
+                      field_type='datetimetz')
+            datetime = self.pull_result(column_name='datetimetz')
+            # if self.database_driver_provides_dates_and_datetimes_as_strings():
+            if isinstance(datetime, str):
+                # TODO
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker